README.md

```diff
@@ -1,24 +1,32 @@
 [<img src="https://github.com/insolar/doc-pics/raw/master/st/github-readme-banner.png">](http://insolar.io/?utm_source=Github)
 
 # Insolar Observer
```

Comment:
This readme's structure needs an overhaul. Requesting changes.

Reply:
Done on a Zoom call